src/commands/verify.ts

```diff
@@ -0,0 +1,50 @@
+
+import { 
+    SlashCommandBuilder,
+    SlashCommandStringOption,
+    GuildMemberRoleManager
+} from "discord.js";
+
+import { CoCommand } from "../structures";
+import axios from "axios";
+
+const Verify = new CoCommand({
+    data: new SlashCommandBuilder()
+        .setName("verify")
+        .setDescription("Verify membership with your email!")
+        .addStringOption((option: SlashCommandStringOption) => 
+        option
+            .setName("email")
+            .setDescription("Your email address")
+            .setRequired(true)
+    ),
+    execute: async ({ interaction })=> {
+        try {
+            await interaction.deferReply({ ephemeral: true });
+
+            const userEmail = interaction.options.get("email")?.value;
+            const discordId = interaction.user.id
+            const verifyUser = await axios.patch(`http://localhost:3000/v1/users/discord/verify?email=${userEmail}&discordId=${discordId}`)
```

Comment:
**refactor:** we should dynamically know when to use the development API url and production API url